tests/fixtures/metadata_factory.py

```diff
@@ -0,0 +1,51 @@
+import factory
```

Comment:
Are we still using this factory? I think those classes were removed, right?

Reply:
I used the `factory.DictFactory` to create a `dict` that follows the structure of these classes. This way, even without the type classes we can generate test cases.